plugins/org.jkiss.dbeaver.ui.editors.hex/src/org/jkiss/dbeaver/ui/editors/binary/pref/HexPreferencesManager.java

```diff
@@ -356,6 +346,24 @@ private void populateFixedCharWidthFontsAsync()
         }
     }
 
+    private boolean isMonospacedFont(@NotNull FontData fontData, boolean isScalable) {
+        int height = 10;
+        if (!isScalable) {
+            height = fontData.getHeight();
+        }
+        Font font = new Font(Display.getCurrent(), fontData.getName(), height, SWT.NORMAL);
+        fontsGc.setFont(font);
+        char ch = ' ';
```

Comment:
Can you please extract these obscure characters (`' '` and `'\u007F'`) to a class constants?